Makefile

```diff
@@ -167,14 +167,25 @@ DOCKER_PUSH_CMD = docker buildx build \
 docker-build-push:
 	$(DOCKER_PUSH_CMD)
 
+.PHONY: docker-build-push
 
 # --- Release -------------------------------------------------------------------
 NEXTTAG := $(shell { git tag --list --merged HEAD --sort=-v:refname; echo v0.0.0; } | grep -E "^v?[0-9]+.[0-9]+.[0-9]+$$" | head -n1 | awk -F . '{ print $$1 "." $$2 "." $$3 + 1 }')
 
+DOCKER_LOGIN = printenv DOCKER_PASSWORD | docker login --username $(DOCKER_USERNAME) --password-stdin
+docker-login:
+	$(DOCKER_LOGIN)
+
+docker-push-hub: REGISTRY = transflect
+docker-push-hub: docker-login
+	$(DOCKER_PUSH_CMD)
+
+release: REGISTRY = transflect
 release:
 	git tag $(NEXTTAG)
 	git push origin $(NEXTTAG)
 	goreleaser release --rm-dist
+	[ -z "$(CI)" ] || $(DOCKER_LOGIN) && $(DOCKER_PUSH_CMD)
```

Comment:
there is a lot of ambiguity in this line. According to the bash man page, `&&` and `||` have the same precedence so it will still run docker push if `$(CI)` is empty (try running `true || false && echo wow`). The pipe (`|`) has higher precedence than `&&` and `||`, so it is safe in the middle (DOCKER_LOGIN contains the pipe), but I had to check that to be sure.

It might be better to write this as:
```
release:
    git tag $(NEXTTAG)
    git push origin $(NEXTTAG)
    goreleaser release --rm-dist
ifneq ($(CI),)
    $(DOCKER_LOGIN)
    $(DOCKER_PUSH_CMD)
endif
```

Reply:
sorry, this was meant to be
```release:
	git tag $(NEXTTAG)
	git push origin $(NEXTTAG)
	goreleaser release --rm-dist
	[ -z "$(CI)" ] || $(DOCKER_LOGIN)
	$(DOCKER_PUSH_CMD)
```